spec/shared_examples/rhel_package_provider.rb

```diff
@@ -125,6 +125,14 @@
         provider.stubs(:query).returns(:ensure => current_version).then.returns(:ensure => version)
         provider.install
       end
+      it 'should run upgrade command if present and ensure latest' do
+        current_version = '1.0'
+        version = '1.2'
+        resource[:ensure] = :latest
+        Puppet::Util::Execution.expects(:execute).with(["/usr/bin/#{provider_name}", '-d', '0', '-e', error_level, '-y', upgrade_command, name])
```

Comment:
Re contributor summit, we discovered that when upgrading a package with `:latest`, puppet will just call `upgrade <package>` *without* the version